src/main.rs

```diff
@@ -124,7 +124,7 @@ pub fn handle_payload(inject_id: String, api: &Client, contract_payload: &Inject
         let executor = contract_payload.payload_cleanup_executor.clone().unwrap();
         let _ = handle_execution_command(
             "cleanup execution",
-            &api,
+            api,
```

Comment:
If I understand correctly, $api allows you to use the $api variable directly without making a copy.
Wouldn't it be more efficient to keep $api everywhere?